config/webpack.vendor.js

```diff
@@ -22,6 +22,8 @@ module.exports = [
   'prop-types',
   'seamless-immutable',
   'isomorphic-fetch',
+  'react-intl',
+  'intl',
```

Comment:
i see jsonp duplicated in many per language bundles, perhaps you need to add ```intl/locale-data/jsonp``` to the list?

Reply:
I checked the bundles and I don't see any dependencies in the language chunks. `intl/locale-data/jsonp` is just path to the files with translations. Webpack visualiser does shows the path components but with 0 size.